src/osmLayer.js

```diff
@@ -1,88 +1,85 @@
-module.exports = (function () {
-  'use strict';
+var $ = require('jquery');
+var inherit = require('./inherit');
+var tileLayer = require('./tileLayer');
+var registry = require('./registry');
+var quadFeature = require('./quadFeature');
 
-  var $ = require('jquery');
-  var inherit = require('./inherit');
-  var tileLayer = require('./tileLayer');
-  var registry = require('./registry');
-  var quadFeature = require('./quadFeature');
+/**
+ * Create a new instance of osmLayer.  This is a [@link geo.tileLayer} with
```

Comment:
Is the @link a typo? I presume it should be _either_ regular brackets or curly brackets

Reply:
It should be braces.  Thanks.